Description: #766994: load cl so cl-macroexpand is available at compile-time
Author: Hilko Bengen <bengen@debian.org>
--- systemtap.orig/emacs/systemtap-mode.el
+++ systemtap/emacs/systemtap-mode.el
@@ -52,6 +52,7 @@
 (require 'cc-mode)
 (require 'cc-awk)
 (eval-when-compile
+  (require 'cl)
   (require 'cc-langs)
   (require 'cc-fonts))
 
